### DISCUSSION TITLE
Building EdgeTX with Qt Creator IDE is not supported anymore

### DISCUSSION THREAD

**Discussion by ViorelX** [EdgeTX/edgetx#2075] [Q&A] [answered]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Build system

### Current Behavior

Hello,
I am trying to build the EdgeTX 2.7.1 firmware by using the Qt Creator IDE. I think this is the most convenient method, since it offers the best way to access and edit the source files.

Last year I was able to build EdgeTX 2.4 up to EdgeTX 2.6, following the instructions found here:
https://github.com/EdgeTX/edgetx/wiki/Building-and-debugging-with-Qt-Creator.

Now it doesn't work anymore. It seems the instructions there are outdated.

### Expected Behavior

I expected to succeed building EdgeTX 2.7.1 very easy, with the Qt Creator IDE already setup for the previous EdgeTX releases.

### Steps To Reproduce

1. Launch the Qt Creator 6.0.1 IDE on Windows 10, 64bits
2. Load the EdgeTX 2.7.1 file system
3. Chose the STM32 kit, properly configured for previous EdgeTX buildings, as target for the firmware
4. Edit the build settings
5. Build: the process enters an infinite loop and should be halted

### Version

2.7.1

### Transmitter

Radiomaster TX16S

### Anything else?

![image](https://user-images.githubusercontent.com/83503278/172191945-4a0b9bcd-4d3d-4fcc-bc87-dd862b119621.png)

**Comment by raphaelcoeffic** [comment, marked as answer]
Unfortunately QT creator does not support sub-cmake very well and takes lots of assumptions on variables, which is why we stopped supporting it.

**Reply by elecpower** [reply]
QT Creator bug report for reference https://bugreports.qt.io/browse/QTCREATORBUG-16113
